lib/logic/child/child_rewards_cubit.dart

```diff
@@ -65,6 +66,7 @@ class ChildRewardsCubit extends ReloadableCubit {
 				Points.fromUICurrency(UICurrency(type: e.type, title: e.title), e.quantity, creator: e.createdBy)).toList()
 			);
 			await _notificationService.sendRewardBoughtNotification(model.id, model.name, child.connections.first, child);
+			// _authBloc.add(AuthenticationActiveUserUpdated(child.copyWith(points: List.from(child.points), rewards: List.from(child.rewards))));
```

Comment:
Is that the only place? What about `CaregiverAwardsCubit`?

Reply:
Oh yeah, adding it there didn't change anything, so I commented this out and then forgot, ops! Should I put it back?

Reply:
I though we had a problem with points not updating here.. is it resolved without this call?

Reply:
The `_authBloc` didn't help and I reverted to updating the widget through its constructor.

Reply:
Ok then, if it works there is probably no point in keeping this here